src/OpenTelemetry.Contrib.Instrumentation.Runtime/Implementation/PerformanceInstrumentation.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="PerformanceInstrumentation.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace OpenTelemetry.Contrib.Instrumentation.Runtime.Implementation
+{
+    internal class PerformanceInstrumentation : IRuntimeInstrumentation
+    {
+        private const string CpuTimeCounterName = "cpu-usage";
+
+        private readonly Meter meter;
+        private readonly IEventCounterStore eventCounterStore;
+        private readonly ObservableGauge<double> cpuTimeCounter;
+        private readonly ObservableGauge<double> workingSetCounter;
+
+        public PerformanceInstrumentation(Meter meter, IEventCounterStore eventCounterStore)
+        {
+            this.meter = meter;
+            this.eventCounterStore = eventCounterStore;
+
+            this.eventCounterStore.Subscribe(CpuTimeCounterName, EventCounterType.Mean);
```

Comment:
The instrumentation classes should probably implement `IDisposable` and clean up subscriptions in the event the `MeterProvider` is shutdown.

Reply:
Great hint. Subscription clean up is now done on Dispose.